packages/arb-token-bridge-ui/tests/e2e/specs/depositCctp.cy.ts

```diff
@@ -3,236 +3,273 @@
  */
 
 import { formatAmount } from '../../../src/util/NumberUtils'
-import {
-  getInitialERC20Balance,
-  getL1TestnetNetworkConfig,
-  getL2TestnetNetworkConfig,
-  zeroToLessThanOneETH
-} from '../../support/common'
+import { zeroToLessThanOneETH } from '../../support/common'
 import { CommonAddress } from '../../../src/util/CommonAddressUtils'
+import { shortenAddress } from '../../../src/util/CommonUtils'
 
 describe('Deposit USDC through CCTP', () => {
-  const USDCAmountToSend = Number((Math.random() * 0.001).toFixed(6)) // randomize the amount to be sure that previous transactions are not checked in e2e
-
   // Happy Path
   context('User has some USDC and is on L1', () => {
-    let l1USDCBal: string
-    let l2USDCBal: string
-    let l2USDCeBal: string
+    let USDCAmountToSend: number = 0
 
     // log in to metamask before deposit
     beforeEach(() => {
+      USDCAmountToSend = Number((Math.random() * 0.001).toFixed(6)) // randomize the amount to be sure that previous transactions are not checked in e2e
```

Comment:
Are we going to have to top up the USDC and ETH here regularly to have the e2e running?

Reply:
Even if we deployed the Circle smart contract, we would still need the API, which is a no-go.

Reply:
But I don't understand why do we randomize the number here, if we create a new wallet every time?

Reply:
We are not creating a new wallet for CCTP right now. The PR - https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1404 fixes this but it needs to be reviewed and go live also. We can remove the randomizing changes in this PR? Currently 1476 blocks bridge-sdk changes hence we prioritized this first.